kalitectl.py

```diff
@@ -345,12 +336,12 @@ def manage(command, args=[], in_background=False):
         utility.prog_name = 'kalite manage'
         utility.execute()
     else:
-        if os.name != "nt":
-            thread = ManageThread(command, args=args, name=" ".join([command] + args))
-            thread.start()
-        else:
-            # TODO (aron): for versions > 0.13, see if we can just have everyone spawn another process (Popen vs. ManageThread)
-            Popen([sys.executable, os.path.abspath(sys.argv[0]), "manage", command] + args, creationflags=CREATE_NEW_PROCESS_GROUP)
+        # Create a new subprocess, it will die together with the main process
```

Comment:
Sorry, that was an old comment from an experiment with `atexit.register` which unfortunately made it die with the wrong parent (after forking) so I had to drop it. I'll fix the comment...